main/src/ca/uwaterloo/flix/api/lsp/provider/CompleteProvider.scala

```diff
@@ -21,10 +21,39 @@ object CompleteProvider {
+    CompletionItem("Float64", "Float64", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Impure", "Impure", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("=")),
+    CompletionItem("Int", "Int", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int8", "Int8", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int32", "Int32", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Int64", "Int64", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("let", "let", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("namespace", "namespace", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("{", "}")),
+    CompletionItem("new", "new", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Nil", "Nil", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("println", "println", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("(", ")")),
+    CompletionItem("pub", "pub", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Pure", "Pure", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List("=")),
+    CompletionItem("ref", "ref", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("rel", "rel", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("select", "select", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("String", "String", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("spawn", "spawn", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("true", "true", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, List(";")),
+    CompletionItem("type", "type", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("Unit", "Unit", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("use", "use", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil),
+    CompletionItem("with", "with", None, Some("flix keyword"), CompletionItemKind.Keyword, InsertTextFormat.PlainText, Nil)
   )
 
```

Comment:
Last comment for the Keywords: I think it is looking good, but please take a careful look at `Parser.scala` to ensure everything is covered. Thanks!